sdk/tables/data-tables/src/TableClient.ts

```diff
@@ -443,13 +443,13 @@ export class TableClient {
     const { span, updatedOptions } = createSpan(`TableClient-upsertEntity-${mode}`, options);
 
     try {
-      if (!entity.PartitionKey || !entity.RowKey) {
+      if (!entity.partitionKey || !entity.rowKey) {
```

Comment:
I think we should update the casing of the property names in the error message on L447 as well.